### PR TITLE
Removing es6 keywords

### PR DESCRIPTION
**Pull request by bhoomipatel6386**
Wondering if I need to update package version

### CODE REVIEW

**Comment by jcbuena** [comment]
Do we also need to convert arrow functions into traditional functions?

File: src/environment.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 'use strict';
 
-const assert = require('chai').assert;
+var assert = require('chai').assert;
 
-const Environment = {
+var Environment = {
   registerEnvironments(environmentMap) {
```
Comment (jcbuena) [inline src/environment.js]: I think we need this to be `registerEnvironments: function(environmentMap)`

**Comment by jcbuena** [comment]
Let's also remove `es6: true` in `.eslintrc` so it explicitly fails if we use ES6.

**Comment by TheSavior** [comment]
I'm curious what caused the need for this change

**Comment by bhoomipatel6386** [comment]
> change

**Comment by bhoomipatel6386** [comment]
@TheSavior this PR is to remove ES6 dependency and thus removing the need for a transpiler

**Pull request merged**